Examples/MAX32662/UART/main.c

```diff
@@ -133,15 +145,23 @@ int main(void)
     MXC_Delay(MXC_DELAY_SEC(1));
 
     // Initialize the UART
+#if defined(BOARD_EVKIT_V1)
     if ((error = MXC_UART_Init(WRITING_UART, UART_BAUD, MXC_UART_APB_CLK, MAP_A)) != E_NO_ERROR) {
+#else
+    if ((error = MXC_UART_Init(WRITING_UART, UART_BAUD, MXC_UART_APB_CLK, MAP_B)) != E_NO_ERROR) {
```

Comment:
#if else condition seems not needed. 
The function and parameters are exactly same.

Reply:
if calls MAP_A else calls MAB_B. They are not exactly same.

Reply:
The UART0B pins are accessible on the EvKit (RX: P0.7; TX: P0.8). Would it be easier if both the EvKit and FTHR boards use UART0B as the WRITING_UART and UART1A as the READING_UART? Unless you want to show the MAP_A vs MAP_B difference, this will minimize the amount of #if,#else,#endif directives used between the two boards if both run on the same set of pins.